app/controllers/katello/api/v2/content_exports_controller.rb

```diff
@@ -0,0 +1,62 @@
+module Katello
+  class Api::V2::ContentExportsController < Api::V2::ApiController
+    before_action :find_exportable_content_view_version, :only => [:export]
+
+    api :GET, "/content_exports/export_histories", N_("Show the export history for a content view version")
+    param :content_view_version_id, :number, :desc => N_("Content view version identifier"), :required => false
+    param :content_view_id, :number, :desc => N_("Content view identifier"), :required => false
+    param :destination_server, String, :desc => N_("Destination Server name"), :required => false
+    param :organization_id, :number, :desc => N_("Organization identifier"), :required => false
+    param :export_history_id, :number, :desc => N_("Content view version export history identifier"), :required => false
+    param_group :search, Api::V2::ApiController
+    add_scoped_search_description_for(ContentViewVersionExportHistory)
+    def export_histories
```

Comment:
would a better  name for this now be `/content_exports/histories` with controller method `histories` now that we are in the 'ContentExportsController` throwing the word 'export' around can be redundant

Reply:
or maybe this now becomes the `index` method. any thoughts @partha ?

Reply:
Changing to `index`